packages/client/lib/util/debug.ts

```diff
@@ -25,7 +25,7 @@ const level = require('level')
 import Common from '@ethereumjs/common'
 import { Block } from '@ethereumjs/block'
 import VM from './lib'
-import { SecureTrie as Trie } from '@ethereumjs/trie'
+import { SecureTrie as Trie } from ''
```

Comment:
It seems like you changed every @ethereumjs/trie to the empty string...

Reply:
Oh, that might have been close as hard as spotting a consensus error. 😂  😂  😂

head -> table, head -> table, head -> table.